### PR TITLE
Routine i18n updates on 20 Sep 2019

### PR DESCRIPTION
**Pull request by BrettDong**
#### Summary
```SUMMARY: I18N "Routine i18n updates on 20 Sep 2019"```

### CODE REVIEW

File: lang/po/zh_CN.po [hunk 1]
```diff
@@ -180183,31 +180182,31 @@ msgstr "一个"
...
-msgstr "在<color_red>流血</color>。"
+msgstr "<color_red>流血</color>了。"
 
 #: src/iuse.cpp
 msgid " looks <color_green>happy</color>. "
 msgstr "看起来<color_green>很快乐</color>。"
 
 #: src/iuse.cpp
 msgid "downed"
-msgstr "倒"
```
Comment (Qrox) [inline lang/po/zh_CN.po]: @cainiaowu Why are this string (and some others below) changed? They were specifically translated that way to make photo descriptions actually make sense in Chinese. (for example, ”某某倒在桌子上“ vs. ”某某击倒在桌子上“，“某某躺在瓦砾上” vs. “某某躺下在瓦砾上”)

File: lang/po/zh_CN.po [hunk 2]
```diff
@@ -181919,12 +181918,12 @@ msgstr "8小时"
...
 #: src/iuse_actor.cpp
 msgid "Until you gain a spell level"
-msgstr "直到你获得法术等级"
+msgstr "直到法术升级"
 
 #: src/iuse_actor.cpp
 #, c-format
 msgid "This item casts %s at level %i."
-msgstr "这件魔道具可以施放%s，等级%i。"
+msgstr "这件魔道具可以施放 %i 级 %s。"
```
Comment (cainiaowu) [inline lang/po/zh_CN.po]: This  one is  problematic since it change argument position which will  cause crash.

File: lang/po/zh_CN.po (hunk 1 above)
Comment (cainiaowu) [reply lang/po/zh_CN.po]: 👌

File: lang/po/zh_CN.po (hunk 2 above)
Comment (BrettDong) [reply lang/po/zh_CN.po]: This should be fixed at Transifex side.

**Pull request closed without merging**